bin/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if [ -f "${DOCKER_CERT_PATH}/ca.pem" ]
```

Comment:
> Kirby was my inspiration for the current approach FWIW

That's fair. The difference between this and the referenced example is that the example was _just_ a shell script, so ContainerPilot executed the shell script, it established its environment, and then all was good. Chaining that to a subsequent app is where it falls apart.

ContainerPilot 3 will have better support for changing the env vars via the control plane, but for now...